src/ralph/data_center/models/physical.py

```diff
@@ -439,6 +443,42 @@ def change_rack(cls, instances, request, **kwargs):
         for instance in instances:
             instance.rack = rack
 
+    @classmethod
+    @transition_action(
```

Comment:
this action should be the only action in single transition - add additional param to check for that.